requirements.txt

```
@@ -1,2 +1,2 @@
-netaddr
-purefusion
+netaddr==0.8.0
+purefusion==1.0.10
```

by pinning to a version you will require a new collection to be published each time a new purefusion module is released. 
Assuming the SDK is always going to be backwards compatible (which it should...) then there is no reason to pin to a version.

---

You're right. But then, we should introduce upper bound, as backward compatibility is guaranteed only in minor and patch versions. Of course, that's the case only if we follow semantic versioning in `purefusion` module (it's not mentioned in Readme there, but I guess we do).
```suggestion
purefusion>=1.0,<2.0
```